### PR TITLE
:adhesive_bandage: filter bots

### PR DESCRIPTION
**Pull request by ras0q**

### CODE REVIEW

File: infrastructure/repository/user_impl.go [hunk 1]
```diff
@@ -116,13 +116,18 @@ func (r *UserRepository) SyncUsers(ctx context.Context) error {
...
 	}
 
-	users := lo.Map(traqUsers, func(u *external.TraQUserResponse, _ int) *model.User {
-		return &model.User{
-			ID:    u.ID,
-			Name:  u.Name,
-			State: u.State,
-		}
-	})
+	users := lo.Map(
```
Comment (H1rono) [inline infrastructure/repository/user_impl.go:119]: `FilterMap`の方が良さそう
https://github.com/samber/lo?tab=readme-ov-file#filtermap

**Comment by ras0q** [comment]
やべ

**Comment by H1rono** [comment]
やべやべ

**Comment by ras0q** [comment]
#660

**Pull request merged**